lib/pliny/commands/updater.rb

```diff
@@ -1,10 +1,11 @@
 require 'fileutils'
 require 'pathname'
 require 'pliny/version'
+require 'pliny/commands/creator'
 require 'uri'
 
 module Pliny::Commands
-  class Updater
+  class Updater < Creator
```

Comment:
I think I did this to avoid some code repetition. I'm fine removing it though, I just did so.